README.md

```diff
@@ -1 +1,27 @@
 # Istio 101
+
+This is the repository for the [Istio][istio] 101 workshop from IBM.
+
+- [presentation](./presentation) -- the data for the teacher/leader of the workshop
+- [workshop](./workshop) -- the vanilla code for the code of the workshop
+
+
+```text
+Copyright:: IBM, Inc 2018-
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
please point to ours - otherwise what's the point of having it in our repo

Reply:
will do, i'll removed the it, and take standardize on what we have in other places.